project/Mima.scala

```diff
@@ -0,0 +1,13 @@
+import com.typesafe.tools.mima.core._
+
+// More details about Mim:
+// https://github.com/typesafehub/migration-manager/wiki/sbt-plugin#basic-usage
+object Mima {
+  val ignoredABIProblems: Seq[ProblemFilter] = {
+    Seq(
+      ProblemFilters.exclude[Problem]("scala.meta.internal.*"),
+      ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.meta.parsers.Parsed.*"),
+      ProblemFilters.exclude[ReversedMissingMethodProblem]("scala.meta.tokenizers.Tokenized.*")
```

Comment:
How do we deal with these issues in the future? Under what conditions do we allow adding new rules here?

Reply:
We use our common sense to determine if these reported ABI issues affect downstream users. In this case I think it's pretty clear that no regular user will be affected.